hermes/probe/delete/delete.go

```diff
@@ -13,9 +13,9 @@
 // limitations under the License.
 //
 // Delete_file implements the probe operation for deleting a file in a
-// storage systemetrics.
+// storage system.
 
-// Package delete implements the file deletion operation with a storage systemetrics.
+// Package delete implements the file deletion operation with a storage system.
```

Comment:
Please verify that GoDoc do the right thing and renders line 18 as the package doc.

Reply:
I'm currently unable to run GoDoc on my code (pinged you with explanation).

Reply:
Ran GoDoc on my personal laptop.

It renders this line as the package doc as expected.